[Refact] #42 twemoji 제거

sewonkimm opened the pull request.

sewonkimm committed:
[Refact] #42 twemoji 라이브러리 제거

- twemoji 렌더링에 소요되는 시간을 줄이기 위해 이모티콘을 image sprite로 만들어 스타일링 하는 방식으로 변경
- 중간중간 제대로 나오지 않는 이모티콘은 카테고리의 불일치 or 이모티콘 코드 불일치 때문 => 추후 변경작업 해줘야함.
- 카테고리 명 'places'를 'place'로 변경
- twemoji 렌더링을 제거했으므로 setGrid, copy 등 메소드 수정 필요

sewonkimm committed:
[Chore] #42 매치되지 않는 이모티콘 수정

sewonkimm committed:
[Refact] #42 skintone 버튼 추가

sewonkimm committed:
[Chore] emoji.json 수정

sewonkimm committed:
[Refact] #42 skintone 버튼 기능 구현

sewonkimm committed:
[Fix] skintone 버튼 클릭 버그 수정

sewonkimm committed:
[Refact] #42 twemoji 라이브러리 제거

- twemoji 라이브러리를 완전히 제거했습니다!(박수 짝짝짝)
- Recent 부분 구현했습니다.

soyoungjeong commented on js/service/getJSON.mjs:
```diff
@@ -0,0 +1,17 @@
+  xhr.send();
+};
+
+export default GET;
```
```
fetch('./file.json')
  .then(response => response.json())
  .then(obj => console.log(obj))
```
저는 이런방식이 더 간단해보이네요~! 

파일이 많지 않다면
```
export default
{
  // my json here...
}
```
```
import emoji from '../emoji.js'

```
이것도 좋아보입니다ㅎㅎ

참고: https://stackoverflow.com/questions/34944099/how-to-import-a-json-file-in-ecmascript-6

soyoungjeong committed:
[Refact] #18 로딩 속도 개선

json 파일을 바로 import하여 데이터 통신 과정을 줄임.
appendchild가 너무 많이 일어나서 속도가 느려지는 것을 없애기 위해
apeendchild하는 부분을 하나로 묶어 innerHTML로 변경.

sewonkimm committed:
Merge pull request #46 from mojimojicorp/Refact/18

Refact/18 로딩속도 개선

sewonkimm committed:
[Chore] json 관련 파일 제거
